packages/promotion/integration-tests/__tests__/services/promotion-module/compute-actions.spec.ts

```diff
@@ -3424,13 +3424,13 @@ describe("Promotion Service: computeActions", () => {
           {
             action: "addShippingMethodAdjustment",
             shipping_method_id: "shipping_method_express",
-            amount: 83.33333333333331,
+            amount: 166.66666666666666,
```

Comment:
q: what happened here?

Reply:
There was a bug in the calculations that was fixed here. It was an incorrect test to begin with. It incorrectly inferred that part of the value was already used.